posts/Fabio/2018-04-16-hibernate-ogm-5-4-Alpha1-released.adoc

```diff
@@ -0,0 +1,68 @@
+= "Hibernate OGM 5.4.0.Alpha1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Alpha1]
+
+Hibernate OGM is now compatible with Hibernate ORM 5.3 and JPA 2.2!
```

Comment:
It's seems a bit too short, I usually add a short description of all the important changes in the first paragraph. This way it also looks good as a abstract. In this case I would mention the the WildFly modules artifact has been replace by a new artifact.

Reply:
Also the fact that it is now possible to run stored procedures with Infinispan Embedded and MongoDB

Reply:
Maybe for MongoDB was already available, and you can also add the next release will have stored procedures for Infinispan Remote as well